### ISSUE TITLE
Returns valid if the secret is close to correct

### ISSUE THREAD

**Issue by itslenny** [kjur/jsrsasign#124]
Using a token I generated using the secret `616161` I get a isValid value of true with any value starting with `616161`.

This is the code I used.

```
var token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdXRob3IiOiJodHRwOi8vd3d3Lml0c2xlbm55c2ZhdWx0LmNvbSIsInNlcmlhbCI6Im1obWhobW5mZ2htaGZnbWZnaG1mZ2htZmdobSIsInByb2R1Y3QiOiJGaW5hbCBNYXJrZG93biIsImNyZWF0ZWRBdCI6MTQ0NDAwMjEwN30.NlYMkd79i0bzP1mJHqpl4ixor9Hr1XS7BF4j5MqICqE';
var isValid = KJUR.jws.JWS.verify(token, "616161", ["HS256"]);
console.log('isValid',isValid);
```

isValid was `true` for the following values:

* 616161 - The actual secret
* 6161616  - added a 6
* 616161a - added an a

I stopped testing, but it seems like you can add any 1 character to the end of the secret and it will validate as true.

**Comment by kjur** [comment]
Thanks for the report. This can be raise because of lacking hexadecimal password checking. I'm planning to change how to provide password by various manner like hexadecimal or utf8 string or base64 string. Then I'll fix also fix this kind of issue. Thanks.

**Comment by kjur** [comment]
I've fixed the issue and released 4.10.0 today. Thanks.
https://github.com/kjur/jsrsasign/releases/tag/4.10.0
